.travis.yml

```diff
@@ -1,21 +1,47 @@
 language: php
+
+services:
+  - elasticsearch
+
 php:
-  - "5.5"
-  - "5.4"
-  - "5.3"
+  - 7.0
+  - 5.6
+  - 5.5
+  - 5.4
+  - 5.3
+  - hhvm
+
+matrix:
+  fast_finish: true
+  allow_failures:
+    - php: 7.0
+    - php: hhvm
+  include:
+    - php: 5.6
+      env: ES_VERSION="1.4.4"
+    - php: 5.6
+      env: ES_VERSION="1.3.9"
+    - php: 5.6
+      env: ES_VERSION="1.2.4"
+    - php: 5.6
+      env: ES_VERSION="1.1.2"
+    - php: 5.6
+      env: ES_VERSION="1.0.3"
+
 env:
-  - ES_VERSION=0.90.0 ES_TEST_HOST=http://localhost:9200
-  - ES_VERSION=0.90.1 ES_TEST_HOST=http://localhost:9200
-  - ES_VERSION=0.90.2 ES_TEST_HOST=http://localhost:9200
-  - ES_VERSION=0.90.3 ES_TEST_HOST=http://localhost:9200
-  - ES_VERSION=0.90.4 ES_TEST_HOST=http://localhost:9200
-  - ES_VERSION=0.90.5 ES_TEST_HOST=http://localhost:9200
+  - ES_VERSION="1.5.1"
+  - ES_TEST_HOST=http://localhost:9200
```

Comment:
this should use the ``env: global:`` config to avoid creating multiple jobs (the short syntax is for env-based matrix)